hack/images

```diff
@@ -60,46 +60,46 @@ imageLegacy() {
 }
 
 imageDocker() {
-	imageDockerCommon $progressFlag -f ./hack/dockerfiles/test.buildkit.Dockerfile
+	imageDockerCommon ${progressFlag} -f ./hack/dockerfiles/test.buildkit.Dockerfile
 }
 
 image() {
 	pushFlag=""
-	if [ "$PUSH" = "push" ]; then
+	if [ "${PUSH}" = "push" ]; then
 		pushFlag="push=true"
 	fi
 
 	tagLatest=""
 	tagLatestRootless=""
-	if [[ "$gitTag" == "$TAG" ]]; then
-		tagLatest=",$REPO:latest"
-		tagLatestRootless=",$REPO:rootless"
+	if [ "${gitTag}" = "${TAG}" ]; then
+		tagLatest=",${REPO}:latest"
+		tagLatestRootless=",${REPO}:rootless"
 	fi
 
 	set -x
 
-	buildctl build $progressFlag --frontend=dockerfile.v0 \
+	buildctl build ${progressFlag} --frontend=dockerfile.v0 \
 		--local context=. --local dockerfile=. \
 		--opt filename=./hack/dockerfiles/test.buildkit.Dockerfile \
-		--opt platform=$PLATFORMS \
-		--output type=image,name=$REPO:$TAG$tagLatest,$pushFlag
+		--opt "platform=${PLATFORMS}" \
+		--output "type=image,name=${REPO}:${TAG}${TAG}Latest,${PUSH}Flag"
```

Comment:
I think this is wrong.
```suggestion
		--output "type=image,name=${REPO}:${TAG}${TAG}Latest,${pushFlag}"
```

Reply:
arf; you're right; looks like I've been sleeping on my latest changes 😓